frontend/pages/beta/model/[modelId]/access-request/schema.tsx

```diff
@@ -32,7 +32,7 @@ export default function NewSchemaSelection() {
       {schemas && !isSchemasLoading && (
         <Container maxWidth='md'>
           <Card sx={{ mx: 'auto', my: 4, p: 4 }}>
-            <Link href={`/beta/model/${modelId}`}>
+            <Link href={`/beta/model/${modelId}?tab=access`}>
```

Comment:
I'm not sure about this - what if someone links you this page? or if you were on a different tab and then hit the "Request Access" button above the tabbed view?